.github/workflows/buildCSharp.yml

```diff
@@ -7,33 +7,50 @@ on:
 jobs:
   build:
     name: "Build and release CSharp SDK"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
 
     steps:
       - name: "Checkout develop Branch"
         uses: actions/checkout@v2
         with:
           ref: develop
-        
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.x'
```

Comment:
Better to specify python 3.8 or it'll use 2.7...

Reply:
Note that all the python/ruby versions are cached on the runners, so setup-python / setup-ruby take like 2 seconds!